WordPress/src/debug/AndroidManifest.xml

```diff
@@ -27,4 +27,15 @@
         tools:ignore="ProtectedPermissions"
         android:name="android.permission.CHANGE_CONFIGURATION"
         />
+
+    <!-- Allows forcing IME for screenshot automation -->
+    <uses-permission
+        tools:ignore="ProtectedPermissions"
+        android:name="android.permission.WRITE_SECURE_SETTINGS"
+        />
+
+    <!-- Allows changing SystemUI demo mode for screenshot automation -->
+    <uses-permission
+            android:name="android.permission.DUMP"
+            tools:ignore="ProtectedPermissions" />
```

Comment:
1. Minor (🔍): The configuration above can be automatically reformatted (see `Reformat Code` action):
```
    <!-- Allows forcing IME for screenshot automation -->
    <uses-permission
        android:name="android.permission.WRITE_SECURE_SETTINGS"
        tools:ignore="ProtectedPermissions" />

    <!-- Allows changing SystemUI demo mode for screenshot automation -->
    <uses-permission
        android:name="android.permission.DUMP"
        tools:ignore="ProtectedPermissions" />
```
2. Minor (🔍): Actually, it seems the whole file can be automatically reformatted as there are some other minor formatting adjustments that can be fixed.
3. Minor (🔍): It is recommended that the `<application ... />` node sits below the `<uses-permission ... />` nodes, actually AS is complaining about that. Also, the `application` node can be reformatted as well:
```

    <application
        android:name=".WordPressDebug"
        android:supportsRtl="true"
        tools:replace="android:name,android:supportsRtl" />
```

Reply:
Fixed in 238d880f421647f4af06c469ef015f5005404c96